packages/adapter-vercel/README.md

```diff
@@ -6,13 +6,19 @@ Adapter for Svelte apps that creates a Vercel app, using a function for dynamic
 
 ```sh
 npm install --save-dev @sveltejs/adapter-vercel
+# or
+yarn add --dev @sveltejs/adapter-vercel
```

Comment:
I think this is going to lead to problems. I believe it will install `latest` instead of `next`. And event if you install `@next` it will lead to issues because it will use a specific version. People should follow the instructions here: https://github.com/sveltejs/kit/blob/master/packages/create-svelte/template/README.md#building